src/wasm/wasm-ir-builder.cpp

```diff
@@ -1783,6 +1791,18 @@ Result<> IRBuilder::makeStringSliceIter() {
   return Ok{};
 }
 
+Result<> IRBuilder::makeContNew(HeapType ct) {
+  if (!ct.isContinuation()) {
+    return Err{"expected continuation type"};
+  }
+  ContNew curr(wasm.allocator);
+  curr.contType = ct;
```

Comment:
No need to set this on `curr` since the `visit` implementation doesn't use it.
```suggestion
```

Reply:
Done